add some limitation

allenwux opened the pull request:
If "Allow access to Azure services" is turned off, some Azure SQL Database features (for example: Data Sync) will not work. @DhruvMsft

azurecla commented:
Hi __@meet-bhagdev__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

allenwux committed:
add some limitation

If "Allow access to Azure services" is turned off, some Azure SQL Database features (for example: Data Sync) will not work.
